File: fw-portal-web/src/main/resources/static/resources/js/dto/ImpostoDto.js
class ImpostoDto{
    _baseCalculo;
    _porcentagemIcms;
    _valorIcms;

    constructor(baseCalculo,porcentagemIcms,valorIcms){
        this._baseCalculo = baseCalculo;
        this._porcentagemIcms = porcentagemIcms;
        this._valorIcms = valorIcms;
    }

    getBaseCalculo(){
        return this._baseCalculo;
    }

    getPorcentagemIcms(){
        return this._porcentagemIcms;
    }
    getValorIcms(){
        return this._valorIcms;
    }
}

File: fw-portal-web/src/main/java/com/faraway/fwportal/dto/ImpostoDto.java
package com.faraway.fwportal.dto;

import java.math.BigDecimal;

import com.faraway.fwportal.model.domain.Imposto;

public class ImpostoDto {

	private BigDecimal baseCalculo;
	private Float porcentagemIcms;
	private BigDecimal valorIcms;

	public ImpostoDto(Imposto imposto) {
		super();
		this.baseCalculo = imposto.getBaseCalculo();
		this.porcentagemIcms = imposto.getPorcentagemIcms();
		this.valorIcms = imposto.getValorIcms();
	}

	public BigDecimal getBaseCalculo() {
		return baseCalculo;
	}

	public Float getPorcentagemIcms() {
		return porcentagemIcms;
	}

	public BigDecimal getValorIcms() {
		return valorIcms;
	}

}
